### PR TITLE
Typo fix and a clarification for NLP tutorial

### PR DESCRIPTION
**Pull request by bojanvujatovic**
@vinid Could you confirm this makes sense?

### CODE REVIEW

**Review by M2Lschool-organizers — APPROVED** [review]
Good catch! 
LGTM, but will wait for @vinid 's approval before merging.

**Comment by vinid** [comment]
Thanks a lot! confirming this!

**Pull request merged**
